docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 # Run Redis and PostgreSQL for testing with a local development server.
 # This is used by make run, which starts cm-service in the foreground.
 
-version: "3"
+# version: "3"
```

Comment:
Go ahead an entirely remove, please, thanks -- we ain't never going back to old `compose`, and if we did, that's what `git` is for :-)